README.md

```diff
@@ -381,47 +396,61 @@ Note: You need to match the above format exactly.
 
 ### Manual provisioning
 
-If you want to provision a sandbox outside of a GitHub pull request, you can do
+If you want to create an instance outside of a GitHub pull request, you can do
 so from the shell:
 
 ```python
-from instance.models.instance import SingleVMOpenEdXInstance
-from instance.tasks import provision_instance
-
-instance = SingleVMOpenEdXInstance.objects.create(
-    sub_domain='dogwood.sandbox',
-    name='Dogwood',
-    fork_name='edx/edx-platform',
-    branch_name='named-release/dogwood',
+from instance.models.openedx_instance import OpenEdXInstance
+
+instance = OpenEdXInstance.objects.create(
+    name='Dogwood sandbox',
+    sub_domain='dogwood',
+    # The rest of the parameters are all optional:
+    email='myname@opencraft.com',
+    openedx_release='named-release/dogwood',
     configuration_version='named-release/dogwood',
-    forum_version='named-release/dogwood',
-    notifier_version='named-release/dogwood',
-    xqueue_version='named-release/dogwood',
-    certs_version='named-release/dogwood',
-    ansible_source_repo_url='https://github.com/edx/configuration.git',
+    configuration_source_repo_url='https://github.com/edx/configuration.git',
+    configuration_extra_settings='',
+    use_ephemeral_databases=False,
```

Comment:
@bradenmacdonald Should this list include *all* settings from `OpenEdXAppConfiguration` that are currently supported?